### PR TITLE
OCPBUGS-8097-re: Updated ELB nomenclature throughout docs

### PR DESCRIPTION
**Pull request by dfitzmau**
[OCPBUGS-8097](https://issues.redhat.com/browse/OCPBUGS-8097)

Version(s):
4.14 through to 4.10

Link to docs preview:
* [AWS account limits-OCD](https://61919--docspreview.netlify.app/openshift-dedicated/latest/osd_planning/aws-ccs.html#aws-limits_aws-ccs)
* [Provisioned AWS Infrastructure-OCD](https://61919--docspreview.netlify.app/openshift-dedicated/latest/osd_planning/aws-ccs.html#ccs-aws-provisioned_aws-ccs)
* [AWS account limits](https://61919--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_aws/installing-aws-account.html#installation-aws-limits_installing-aws-account)
* [Provisioned AWS Infrastructure-ROSA](https://61919--docspreview.netlify.app/openshift-rosa/latest/rosa_planning/rosa-sts-aws-prereqs#rosa-aws-policy-provisioned_rosa-sts-aws-prereqs)

QE review:
- [x] QE has approved this change.

Additional information:
* [OSDOCS-5440](https://github.com/openshift/openshift-docs/pull/55321)

### CODE REVIEW

File: modules/dedicated-configuring-your-application-routes.adoc
Comment (dfitzmau) [inline modules/dedicated-configuring-your-application-routes.adoc:8]: In OCP Dedicated 3 docs but not 4.

File: modules/aws-limits.adoc
Comment (codymant) [inline modules/aws-limits.adoc:53]: @dfitzmau The following comments are derived from AWS style guidance. 

On first usage of an AWS service name within a topic, AWS style recommends writing out the full name to introduce the acronym (e.g "Elastic Load Balancing (ELB)"). The acronym may be used on subsequent usage, once the full service name is introduced. 

When referring to specific load balancer types, AWS uses the following terminology. These words are always used together in the phrasing seen below, and written in all caps. 

- Classic Load Balancer 
- Application Load Balancer 
- Network Load Balancer 

See: 

https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html
https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html
https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html

Comment (codymant) [reply modules/aws-limits.adoc:53]: When referring to a generic load balancer and type does not matter (i.e. Classic, Application, or Network), the term "load balancer" is used.

Comment (codymant) [reply modules/aws-limits.adoc:53]: I recommend revising as follows: 

"By default, each cluster creates internal and external ELB Network Load Balancers for the primary API server and a single Classic Load Balancer for the router."

File: modules/aws-limits.adoc
Comment (codymant) [inline modules/aws-limits.adoc:66]: @dfitzmau I would consider dropping the "ELB" here. Earlier in the topic, it's already been introduced that this information applies to ELB resources. The acronym seems unnecessary.

File: modules/dedicated-configuring-your-application-routes.adoc
Comment (codymant) [inline modules/dedicated-configuring-your-application-routes.adoc:8]: @dfitzmau AWS does not use the term "AWS elastic load balancing". The full service name is always "Elastic Load Balancing" by itself, without "AWS". The name should be capitalized also. 

Suggested revision: "When your cluster is provisioned, an Elastic Load Balancing  (ELB) load balancer is created.

File: modules/installation-aws-limits.adoc
Comment (codymant) [inline modules/installation-aws-limits.adoc:72]: @dfitzmau The heading above intros the service name, so there's no need to repeat here. Suggested revision: "single Classic Load Balancer for the router."

File: modules/rosa-aws-provisioned.adoc
Comment (codymant) [inline modules/rosa-aws-provisioned.adoc:90]: @dfitzmau Should say "Amazon EC2 instance(s)", instead of "EC2 instance".

File: modules/aws-limits.adoc
Comment (dfitzmau) [reply modules/aws-limits.adoc:53]: Thanks, @codymant . I updated the sentence based on your feedback. I updated the component name, `ELB/NLB`, to `Elastic Load Balancing (ELB)`. I removed `NLB` as it seemed to confuse things. WDYT?

**Comment by yunjiang29** [comment]
@lihongan can you take care of this PR?

File: modules/aws-limits.adoc
Comment (lihongan) [inline modules/aws-limits.adoc:53]: I think `each cluster creates internal and external Network Load Balancers for the primary API server ...` is OK, no need to mention ELB here.

File: modules/ccs-aws-provisioned.adoc
Comment (lihongan) [inline modules/ccs-aws-provisioned.adoc:49]: `Elastic Load Balancing (ELB) load balancers` is better, please capitalize L and B

**Comment by lihongan** [comment]
other looks good. thank you!

File: modules/aws-limits.adoc
Comment (dfitzmau) [reply modules/aws-limits.adoc]: Thanks. Updated.

File: modules/ccs-aws-provisioned.adoc
Comment (dfitzmau) [reply modules/ccs-aws-provisioned.adoc]: Well spotted. Updated.

**Review by skopacz1 — COMMENTED** [review]
PR looks good! I just left a few comments. Please wait until @adellape approves my review before implementing anything, thanks!

**Review by adellape — COMMENTED** [review]
@skopacz1 Peer review comments LGTM 👍 

@dfitzmau Also please note in comment 0 which versions to cherry-pick this to.

File: modules/rosa-aws-provisioned.adoc
Comment (dfitzmau) [reply modules/rosa-aws-provisioned.adoc:90]: Thanks. Option B sounds good to me.

File: modules/installation-aws-limits.adoc
Comment (dfitzmau) [reply modules/installation-aws-limits.adoc:1]: I checked this file and do not see the "ELB/NLB" reference you discuss. Are you viewing the latest commit?

File: modules/aws-limits.adoc
Comment (dfitzmau) [reply modules/aws-limits.adoc:1]: Good catch. Updated.

**Comment by dfitzmau** [comment]
Hi @skopacz1 and @adellape . Thank you for the reviews. I updated the PR based on your feedback. I reached out to Cody (an AWS writer) and I dropped the BLN and CLB abbreviations as they are not official abbreviations. See/read:

- https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html
- https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html

**Review by jeana-redhat — APPROVED** [review]
LGTM

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by jeana-redhat** [comment]
@dfitzmau 4.10&4.11 need a manual CP, sorry :bow:

**Pull request merged**
